### PR TITLE
MULTIARCH-4704: Duplicate the prow config to target v0.9. Changed the…

### PR DESCRIPTION
**Pull request by AnnaZivkovic**
… references to registry.ci.openshift.org/origin/multiarch-tuning-operator:v0.9

### CODE REVIEW

File: ci-operator/config/openshift/multiarch-tuning-operator/openshift-multiarch-tuning-operator-v0.9.yaml [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: registry.ci.openshift.org/origin/multiarch-tuning-operator:v0.9
+    with: pipeline:multiarch-tuning-operator
+promotion:
+  to:
+  - excluded_images:
+    - operator-sdk-src
+    namespace: origin
+    tag: main
```
Comment (aleskandro) [inline ci-operator/config/openshift/multiarch-tuning-operator/openshift-multiarch-tuning-operator-v0.9.yaml:44]: This should be v0.9 right?

File: ci-operator/config/openshift/multiarch-tuning-operator/openshift-multiarch-tuning-operator-v0.9.yaml [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+      PROJECT_NAME: multiarch-tuning-operator
+    workflow: openshift-ci-security
+- as: bundle
+  cluster: build10
+  commands: unset VERSION; NO_DOCKER=1 make bundle-verify
+  container:
+    from: operator-sdk-src
+  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Za-z]+\.md$)|((^|/)\.github/)|((^|/)\.tekton/)
+zz_generated_metadata:
+  branch: v0.9
```
Comment (AnnaZivkovic) [inline ci-operator/config/openshift/multiarch-tuning-operator/openshift-multiarch-tuning-operator-v0.9.yaml:243]: This one should not be main right?

Comment (aleskandro) [reply ci-operator/config/openshift/multiarch-tuning-operator/openshift-multiarch-tuning-operator-v0.9.yaml:243]: no and that one will be updated automatically. 

You'd run `make update` to generate the other files

**Review by aleskandro — APPROVED** [review]

**Pull request merged**